syntax = "proto3";
// python3 -m grpc_tools.protoc -I ./ --python_out=./ --grpc_python_out=./ message.proto
 
// сервер, который необходимо поднять 
service MsgService {

//  rpc GetMsg (MsgRequest) returns (stream MsgResponse){}

//   GetMsg - метод реализованный на клиенте, которые отправляет запрос и возвращает ответ
  rpc GetMsg (MsgRequest) returns (MsgResponse){}
  rpc Send (SendRequest) returns (MsgResponse){}

}

 
// запрос клиента
message MsgRequest {

  string name = 1; // то что должен отправить клиент
  int32 number = 2;

}

 
// ответ от сервера
message MsgResponse {

  int32 status = 1; // то что должен отметить сервер, после запроса
  string details = 2;
  

}

message SendRequest {
  string tableName = 1;
  string base64Str = 2;
  string sourceId = 3;
}

